src/test/java/bio/terra/app/controller/UnauthenticatedApiControllerTest.java

```diff
@@ -0,0 +1,76 @@
+package bio.terra.app.controller;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import bio.terra.app.configuration.OauthConfiguration;
+import bio.terra.app.configuration.OpenIDConnectConfiguration;
+import bio.terra.app.configuration.SamConfiguration;
+import bio.terra.app.configuration.TerraConfiguration;
+import bio.terra.model.RepositoryStatusModel;
+import bio.terra.service.job.JobService;
+import bio.terra.service.status.StatusService;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+
+@ActiveProfiles({"google", "unittest"})
+@ContextConfiguration(classes = UnauthenticatedApiController.class)
+@Tag("bio.terra.common.category.Unit")
+@WebMvcTest
+public class UnauthenticatedApiControllerTest {
```

Comment:
Test category: controller unit test which talks to a `MockMvc`.

We opted in the controller we're testing via `@ContextConfiguration`.